pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -222,6 +222,9 @@ self: super: {
 
   });
 
+  # The project by definition uses the network for tests.
+  hnix-store-remote = dontCheck super.hnix-store-remote;
```

Comment:
Not really needed - the tests are disabled by default due to their requirements (which and nix binaries) but don't need network access. All testing is done in namespace using temporary nix store and local nix-daemon.

Reply:
Great info. I thought the tests are effectful to the `/nix/store`.

Anyway, this PR would probably wait for a while.

Please feel free to notify me or open parallel PR, on yours decision.